### PR TITLE
Cleaning house on the backend.

### PR DESCRIPTION
**Pull request by Chris-Hesterman**
Deleting unused code, restructuring to better follow MVC, not done yet.
Need to separate controllers and routes.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -134,33 +135,40 @@ server.put('/reviews/:reviewId', (req, res) => {
...
+app.delete('/reviews/:reviewId', (req, res) => {
+  pgdb
+    .deleteReview(req.params.reviewId)
+    .then((result) => {
+      res.status(200).send(result);
+    })
+    .catch((err) => {
+      res.status(500).send(err);
+    });
 });
```
Comment (samasastudio) [inline server/index.js:148]: heyooo lookin' good!

File: server/index.js [hunk 2]
```diff
@@ -134,33 +135,40 @@ server.put('/reviews/:reviewId', (req, res) => {
...
 
-server.get('/product', (req, res) => {
+app.get('/product', (req, res) => {
   const { itemID } = req.query;
```
Comment (samasastudio) [inline server/index.js:150]: it's funny how even this change makes me feel much better in my soul.

**Review by samasastudio — APPROVED** [review]
Clean!

File: server/index.js (hunk 2 above)
Comment (Chris-Hesterman) [reply server/index.js:150]: I know!!!

**Pull request merged**